.github/workflows/uncrustify.yml

```diff
@@ -14,17 +14,23 @@ jobs:
         env:
           GITHUB_CONTEXT: ${{ toJson(github) }}
         run: echo "$GITHUB_CONTEXT"
+      - name: Install Utils
```

Comment:
As ubuntu:18.04 is not available on Github CI anymore we might need to use docker image of it:
```
    runs-on: ubuntu-20.04
    container: ubuntu:18.04
```